src/erlang_ls_client.erl

```diff
@@ -140,19 +144,21 @@ workspace_symbol(Query) ->
 %%==============================================================================
 %% gen_server Callback Functions
 %%==============================================================================
--spec init({hostname(), port_no()}) -> {ok, state()}.
-init({Host, Port}) ->
+-spec init({tcp, hostname(), port_no()}) -> {ok, state()}.
```

Comment:
Why not to define a `client` behaviour and have a `tcp_client` and a `stdio_client`?

Reply:
Becuase it would have involved a lot more time than I had already invested 😄.